docker-compose-keycloak-mysql.yml

```diff
@@ -0,0 +1,160 @@
+version: "3.7"
+
+x-logging:
```

Comment:
IHMO it is better to keep this outside the docker-compose file , to let people adjust log seeting according to their env

Reply:
I have been thinking about the docker-compose-keycloak-mysql.yml file and it is probably better for me to remove this file and submit a PR later to add it as an example with code and do the following:
1) Rename the existing example directory to examples
2) Move the existing example files into an examples/web_app_simple directory
3) Create another example directory like examples/web_app_complex that uses the docker-compose-keycloak-mysql.yml  to show how to setup and use keycloak with a MYSQL server